src/DataProtection/readme.md

```diff
@@ -0,0 +1,134 @@
+<!-- region Generated -->
+# Az.DataProtection
+This directory contains the PowerShell module for the DataProtection service.
+
+---
+## Status
+[![Az.DataProtection](https://img.shields.io/powershellgallery/v/Az.DataProtection.svg?style=flat-square&label=Az.DataProtection "Az.DataProtection")](https://www.powershellgallery.com/packages/Az.DataProtection/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.2.3 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.DataProtection`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+# My API 
+
+This file contains the configuration for generating My API from the OpenAPI specification.
+
+> see https://aka.ms/autorest
+
+``` yaml
+# it's the same options as command line options, just drop the double-dash!
+require:
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2021-02-01-preview/dataprotection.json
```

Comment:
Replace the master with the commit id to pin the swagger version

Reply:
okay. will do